lib/flutter_calendar_carousel.dart

```diff
@@ -685,7 +683,8 @@ class _CalendarState extends State<CalendarCarousel> {
 
   List<Widget> _renderWeekDays() {
     List<Widget> list = [];
-    for (var weekDay in widget.weekDays) {
+    for (var i = firstDayOfWeek; true; i = (i + 1) % 7) {
```

Comment:
I'll take a look at it tomorrow but it should work. I kind of just copy/pasted how flutter/material.dart generated `showDatePicker` (since I was looking at how they managed the i18n) and adapted it to the situation 😛 .